enable-vcp-image/install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#### ---- Install Package Dependencies ---- ####
+apt-get update && apt-get install -y curl git jq
+rm -rf /var/lib/apt/lists/*
+
+export DEBIAN_FRONTEND="noninteractive"
+export GOVERSION="1.8.3"
+export GOROOT="/opt/go"
+export GOPATH="/root/.go"
+
+cd /opt
+curl -LO https://storage.googleapis.com/golang/go${GOVERSION}.linux-amd64.tar.gz
+tar zxf go${GOVERSION}.linux-amd64.tar.gz && rm go${GOVERSION}.linux-amd64.tar.gz
+ln -s /opt/go/bin/go /usr/bin/
+
+mkdir $GOPATH
+go get github.com/vmware/govmomi/govc
```

Comment:
We can install the latest govc version from source if we have the Go toolchain installed using `go get`
https://github.com/vmware/govmomi/tree/master/govc#source

Do you want to find the latest govc release version and build URL and use the CURL?
This looks easier and straightforward to me.